castle/cms/media/video.py

```diff
@@ -35,11 +35,11 @@ def process(context):
 
     # by default, assume all non-mp4 videos need to be converted
     # but in reality, all videos need converting, even mp4.
-    # md5 is only what makes this possible
```

Comment:
seems like the specific hash wouldn't matter here, just that hashing is used. but this comment could be read like md5 specifically is what makes this particular thing possible -- am I missing anything about the specific use of md5 with the logic here? does it interact with the video conversion in any way?

Reply:
I removed the 2 bottom lines of the comments. After looking over the code I decided they don't really make sense since it say that all videos need converting, even mp4s, but it explicitly does not convert mp4s and I can find nowhere in the code where it handles the conversion of mp4s after the `process` method